docs/taskfile_versions.md

```diff
@@ -15,21 +15,7 @@ available, but not `3.0.0+`.
 
 ## Version 1
 
-In the first version of the `Taskfile`, the `version:` key was not available,
```

Comment:
Actually, I think we should probably still keep this documentation, but add a notice right after the title informing that this is not available anymore on Task >= v3.0.0.